### DISCUSSION TITLE
FLOPs Calculation of espnet model

### DISCUSSION THREAD

**Discussion by bobbiaditya** [espnet/espnet#5499] [Q&A] [answered]
Hi, is there any function that available in espnet to see the FLOPs calculation of a model?

Thanks!

**Comment by sw005320** [comment, marked as answer]
@pyf98, can you answer it?
You seem to have some tools https://github.com/pyf98/espnet_utils

**Comment by pyf98** [comment]
As @sw005320 shared, I used DeepSpeed's profiler to calculate FLOPs. You can check that repo for an example usage. Note that if you have some special operations that are not defined in DeepSpeed, it will not be counted.

**Comment by bobbiaditya** [comment]
thanks for the answer, I will try to use the tools provided
